config/config.py.template

```diff
@@ -143,7 +143,7 @@ COMMON_JAVA_OPTS = [
     # Fraction of JVM memory used for caching RDDs.
     JavaOptionSet("spark.storage.memoryFraction", [0.66]),
     JavaOptionSet("spark.serializer", ["org.apache.spark.serializer.JavaSerializer"]),
-    JavaOptionSet("spark.executor.memory", ["9g"]),
+    # JavaOptionSet("spark.executor.memory", ["9g"]),
```

Comment:
@pwendell We usually run this on spark-ec2 clusters, where an appropriate value of `spark.executor.memory` is provided through `spark-defaults.conf`.  If we leave this `9g` default in here, then it will take precedence over the cluster's setting.

@nchammas We need to ensure that `spark.executor.memory` is set somewhere, since in general I don't think we can auto-detect the right executor size (this value is actually used as part of the request for executors from the cluster resource manager).  The default, if unset, is 512m, which is too small in most cases.